Userland/Libraries/LibC/arch/i386/crti.S

```diff
@@ -0,0 +1,15 @@
+/*
```

Comment:
`arch` -> `Arch` to match the other arch-specific dirs. :)

Reply:
The other subdirectories are in lower case though, and this `arch` directory is also used by other stuff.

Reply:
Oh right, nevermind then.